tools/packaging/kata-deploy/runtimeclasses/kata-clh.yaml

```diff
@@ -0,0 +1,13 @@
+---
+kind: RuntimeClass
```

Comment:
That's a really really good idea @fidencio . Question: why we still need the kata-runtimeClasses.yaml file? Forgot to delete?

Reply:
Because users may still want to have control and have the old behaviour.  It's there for "backwards compat", let's say.